src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -122,8 +108,8 @@
       {TRIPLEA_GAME_PROPERTY, TRIPLEA_SERVER_PROPERTY, TRIPLEA_CLIENT_PROPERTY, TRIPLEA_HOST_PROPERTY,
           TRIPLEA_PORT_PROPERTY, TRIPLEA_NAME_PROPERTY, TRIPLEA_SERVER_PASSWORD_PROPERTY, TRIPLEA_STARTED,
           TRIPLEA_LOBBY_PORT_PROPERTY,
-          LOBBY_HOST, LOBBY_GAME_COMMENTS, LOBBY_GAME_HOSTED_BY, TRIPLEA_ENGINE_VERSION_BIN, HttpProxy.PROXY_HOST,
```

Comment:
We do not run bots or lobby with a proxy arg, given it can be updated via UI, we really do not need command line args for it. Simplifying here by removing them. 

Proxy configuration was stored in the engine preferences window, to move this to the settings window required for me to go through the proxy code. There are some further updates/simplfications to the proxy code